gtsfm/frontend/verifier/verifier_base.py

```diff
@@ -20,56 +20,33 @@ class VerifierBase(metaclass=abc.ABCMeta):
     geometrically verified points.
     """
 
-    def __init__(self, min_pts):
-        self.min_pts = min_pts
+    def __init__(self, min_pts_e_matrix: int, min_pts_f_matrix: int):
+        self.min_pts_e_matrix = min_pts_e_matrix
+        self.min_pts_f_matrix = min_pts_f_matrix
 
     @abc.abstractmethod
-    def verify_with_exact_intrinsics(
+    def verify(
         self,
         keypoints_i1: Keypoints,
         keypoints_i2: Keypoints,
         match_indices: np.ndarray,
         camera_intrinsics_i1: Cal3Bundler,
         camera_intrinsics_i2: Cal3Bundler,
+        use_intrinsics_in_verification: bool = False,
```

Comment:
should this be a property of the verifier, and not an argument of the method?

I guess I dont foresee a situation where we would have some intrinsics but not all (mixed), but perhaps it could arise